docs/reference/settings/security-settings.asciidoc

```diff
@@ -855,6 +855,12 @@ The maximum amount of skew that can be tolerated between the IdP's clock and the
 {es} node's clock.
 Defaults to `3m` (3 minutes).
 
+`req_authn_context_class_ref`::
+A comma separated list of Authentication Context Class Reference values to be
+included in the Requested Authentication Context when requesting the IdP to
+authenticate the current user. The Authentication Context of the corresponding
+authentication response should contain at least one of the requested values.
```

Comment:
Does this need a link to the more complete explanation?
It doesn't make sense on its own unless you're a hard-core SAML expert.

Reply:
I added a link